src/Ironclad/Sdk/AuthCookieMiddleware.cs

```diff
@@ -56,5 +66,34 @@ await handler.HandleRequestAsync())
 
             await this.next.Invoke(context);
         }
+
+        private bool UserUsesLatestIos(HttpContext context)
```

Comment:
This will not always be the latest iOS version. Maybe this should be changed to something else?
```suggestion
        // LINK: https://github.com/IdentityServer/IdentityServer4/issues/2595#issuecomment-425068595
        private bool RequiresSameSiteCookieFix(HttpContext context)
```

Reply:
Any suggestions? `UserUsesLatestIosVersions`

Reply:
`RequiresSameSiteCookieFix` as per the suggestion above?